tests/api/db/test_functions.py

```diff
@@ -320,3 +300,61 @@ def test_delete_function(db: DBInterface, db_session: Session):
 
     assert number_of_tags == 0
     assert number_of_labels == 0
+
+
+@pytest.mark.parametrize(
+    "db,db_session", [(dbs[0], dbs[0])], indirect=["db", "db_session"]
+)
```

Comment:
lets add the comment as we have in the other tests that are skipping fileDB, this will help us in future deprecation of the fileDB